documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -169,6 +171,66 @@ spec:
         sourceWDTInstallHome: None
 ```
 
+### Automated upgrade of WKO 3.x auxiliary images configuration
+We have enhanced the auxiliary images feature in Operator version 4.0 to improve the ease of use, and its configuration has changed from the Operator 3.x releases. 
+
+The 4.0 Operator provides a seamless upgrade of the Domains with WKO 3.x (V8 schema) auxiliary images configuration. When you create a Domain with WKO 3.x (V8 schema) style auxiliary images configuration in a namespace managed by the WKO 4.0, the [WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}}) performs an automated upgrade of the domain resource to 4.0 schema. The conversion webhook runtime converts the WKO 3.x auxiliary image configuration to the equivalent configuration using init-containers, volume and volume mounts under the `serverPod` section in WKO 4.0. Similarly, when [upgrading the Operator version from 3.x to 4.0]({{< relref "/userguide/managing-operators/installation#upgrade-the-operator" >}}), Domains with auxiliary images are seamlessly upgraded.
```

Comment:
Not every customer will understand the reference to "V8 schema". Here's how Kubernetes describes an API version upgrade, e.g. for the Ingress resource: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122:

```
Ingress

The extensions/v1beta1 and networking.k8s.io/v1beta1 API versions of Ingress is no longer served as of v1.22.

* Migrate manifests and API clients to use the networking.k8s.io/v1 API version, available since v1.19.
* All existing persisted objects are accessible via the new API
* Notable changes:
     * spec.backend is renamed to spec.defaultBackend
     * The backend serviceName field is renamed to service.name
     * Numeric backend servicePort fields are renamed to service.port.number
     * String backend servicePort fields are renamed to service.port.name
        pathType is now required for each specified path. Options are Prefix, Exact, and ImplementationSpecific. To  match the undefined v1beta1 behavior, use ImplementationSpecific.
```

I think that we need something like this that talks about the "weblogic.oracle/v8" and "weblogic.oracle/v9" API versions and their notable differences.

Reply:
Changed the wording to use `weblogic.oracle/v8` and `weblogic.oracle/v9` instead of V8/V9 schema.